src/CNEditor.js

```diff
@@ -205,7 +209,13 @@ export default class CNEditor extends Component {
             this.webViewRef.postMessage(jsonString);
         } 
     }
-
+    
+    onValueChanged = (data) => {
+        this.props.initialHtml = data;
```

Comment:
it is not valid to change props. so I will remove this line.
also we use 'initialHtml' just once on load , so changing it has no effect.